### PR TITLE
add review comment functionality

### PR DESCRIPTION
**Pull request by annakrystalli**
### Summary

I've tried to implement comments under reviews for authors to be able to interact with reviewers.

I've followed the instructions here: https://www.samuelliedtke.com/blog/implement-comment-system-blog-application-django/

I can add comments through the admin interface but not through the `review_detail` template. I get this error when I try and post:
```
WARNING 2021-10-08 08:14:36,271 base 35356 123145593565184 Method Not Allowed (POST): /review/13/
Method Not Allowed: /review/13/
WARNING 2021-10-08 08:14:36,271 log 35356 123145593565184 Method Not Allowed: /review/13/
[08/Oct/2021 08:14:36] "POST /review/13/ HTTP/1.1" 405 0
```

Any chance you could help me out @twinkarma ? It should be really close but it seems not to be implementing post correctly!

### CODE REVIEW

**Comment by annakrystalli** [comment]
I've broken the tests now! 😵‍💫

I think it's because of the migration sequences?

File: reprohack_hub/forms.py [hunk 1]
```diff
@@ -382,3 +382,15 @@ def __init__(self, *args, **kwargs):
...
+class CommentForm(ModelForm):
+    class Meta:
+        model = Comment
+        fields = ['comment']
+
+    def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+        super(CommentForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper(self)
+        self.helper.form_method = 'post'
```
Comment (twinkarma) [inline reprohack_hub/forms.py:395]: Need to add the following line:
self.helper.form_action = `whaterver your ReviewComment view's url is`

Comment (annakrystalli) [reply reprohack_hub/forms.py:395]: I thought the `self.helper.add_input` took care of the form action?

**Comment by twinkarma** [comment]
Also noticed in your `reprohack_hub/urls.py` there's no endpoint/url that links to the `ReviewComment` view. 

The process should be:

* When go to ReviewDetail  view, they see a comments form that they can edit
* The comment form submits to ReviewComment view, this adds the comment to the database
* The ReviewComment view, on success, returns the user to the ReviewDetail view

File: reprohack_hub/forms.py (hunk 1 above)
Comment (twinkarma) [reply reprohack_hub/forms.py:395]: Ok I get it now, ignore above comment

**Pull request closed without merging**